app/src/openxr/cpp/OpenXRLayers.h

```diff
@@ -183,7 +175,12 @@ class OpenXRLayerBase : public OpenXRLayer {
   }
 
 protected:
-  virtual ~OpenXRLayerBase() {}
+  ~OpenXRLayerBase() {
```

Comment:
This must be virtual, see [here](https://wiki.sei.cmu.edu/confluence/display/cplusplus/OOP52-CPP.+Do+not+delete+a+polymorphic+object+without+a+virtual+destructor) for more info